functions/submission-created.js

```diff
@@ -0,0 +1,60 @@
+const sgMail = require('@sendgrid/mail');
+const { SENDGRID_API_KEY, ANGKLUNG_EMAIL } = process.env;
+
+// setup sendgrid
+sgMail.setApiKey(SENDGRID_API_KEY);
+
+exports.handler = async function (event, context, callback) {
+    const { name, email, message, formType } = JSON.parse(
+        event.body
+    ).payload.data;
+
+    console.log(`Received contact form submission from ${email}`);
+
+    // set up email msg. We are sending from our self to ourself.
```

Comment:
```suggestion
    // set up email msg. We are sending from ourself to ourself.
```

Reply:
Probably will need to change the `to` email to hello@<our domain>